src/windows/video_widget.py

```diff
@@ -286,12 +298,12 @@ def update_title(self):
 
         if self.settings.get("preview-fps"):
             # Update window title with FPS output
-            dock.setWindowTitle(_("Video Preview") + " " + _("(Paint: %d FPS, Render: %d FPS, %dx%d)")
+            dock.setWindowTitle(_("Video Preview") + speed_str + _("(Paint: %d FPS, Render: %d FPS, %dx%d)")
```

Comment:
Let's move this into the parenthesis, for consistency (when the debug FPS is displayed)